include/ddc/kernels/splines/spline_evaluator_2d.hpp

```diff
@@ -80,13 +85,13 @@ class SplineEvaluator2D
 private:
     spline_domain_type m_spline_domain;
 
-    LeftExtrapolationRule1 m_left1_bc;
+    LeftExtrapolationRule1 m_left_extrap_rule_1;
 
     RightExtrapolationRule1 m_right1_bc;
```

Comment:
Do you mind also renaming `m_right1_bc` ?

Reply:
Didn't realise I'd missed this one. Fixed 77ade06